item-spec/examples/landsat8-sample.json

```diff
@@ -75,7 +75,6 @@
     "metadata": {
       "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_MTL.txt",
       "type": "mtl",
-      "role": "metadata",
```

Comment:
What is this change about?

Reply:
'role' is replaced by 'roles' a few lines down